lib/armrest/auth.rb

```diff
@@ -41,8 +48,16 @@ def msi_credentials
     end
 
     def cli_credentials
-      return unless File.exist?("#{ENV['HOME']}/.azure/accessTokens.json")
       Armrest::Api::Auth::CLI.new(@options)
     end
+
+    def get_access_token
+      command = "az account get-access-token -o json"
+      logger.debug "command: #{command}"
+      out = `#{command}`
+      JSON.load(out)
+    rescue
```

Comment:
Thinking let's rescue a more specific exception. Probably `JSON::ParserError` in case the `JSON.load` bombs. 

Also, in Ruby, the backtick operator to execute command always continues regardless of the system command error. You can check for the command error though via the special variable `$?`. IE: `$?.success?` and also decide whether to raise a `CliError` in that case.

Reply:
Should be fixed with https://github.com/boltops-tools/armrest/pull/4/commits/2b9a548fbb9aafb91c5a21170e2a242ac3b99186